### DISCUSSION TITLE
Laptop stock checker?

### DISCUSSION THREAD

**Discussion by thetechguy-cpu** [jef/streetmerchant#2689] [General]
Hi.
Is it possible to check laptop stocks using this? If so how? In particular, I'm looking at [this](https://www.lenovo.com/us/en/laptops/legion-laptops/legion-5-series/Lenovo-Legion-5-15ARH05/p/88GMY501444)

**Comment by Mr-Tech-13** [comment]
https://www.jef.buzz/streetmerchant/faq/#how-do-i-add-a-store

**Comment by thetechguy-cpu** [comment]
Hi there.
So I've decided to add the store, however I always get this error. 
>  [2:26:42 PM] error ::   [lenovo] lenovo legion5 legion - Navigation timeout of 5000 ms exceeded                                                                  

How can I fix this?
Here is my store.ts file:

```
import {Store} from './store';
import fetch from 'node-fetch';

export const lenovo: Store = {
  backoffStatusCodes: [403, 429, 503],
  currency: '$',
  labels: {
    inStock: {
      container: '#addToCartButtonTop',
      text: ['add to cart'],
    },
    outOfStock: {
      container: '#tab-customize > div.tabbedBrowse-productListings-wrapper > div.tabbedBrowse-productListings-scroller > ol > li:nth-child(1) > div.tabbedBrowse-productListing > div.tabbedBrowse-productListing-footer > div.tabbedBrowse-productListing-button-container.second-button-container > div > button',
      text: ['Add to cart'],
    },
  },
  links: [
    {
      brand: 'lenovo',
      model: 'legion',
      series: 'legion5',
      url: 'https://www.lenovo.com/us/en/laptops/legion-laptops/legion-5-series/Lenovo-Legion-5-15ARH05/p/88GMY501444',
    },

  ],
  name: 'lenovo'
  };
```

**Comment by Mr-Tech-13** [comment]
Go to dotenv and change PAGE_TIMEOUT= to 0

**Reply by thetechguy-cpu** [reply]
> Go to dotenv and change PAGE_TIMEOUT= to 0

Thanks. This seemed to solve the problem.

**Comment by Mr-Tech-13** [comment]
if its not too much trouble, can you send me a google drive link to your bot so I can track that laptop as well?

**Reply by thetechguy-cpu** [reply]
> if its not too much trouble, can you send me a google drive link to your bot so I can track that laptop as well?

Hi.
You can just copy the lenovo.ts file I posted above then follow the rest of this [guide](https://www.jef.buzz/streetmerchant/help/general/#adding-a-store)
